### PR TITLE
Created Mobile Navigation

### PR DESCRIPTION
**Pull request by stanislav-slavov**
ref #258

### CODE REVIEW

File: src/components/layout/MobileNavigation.vue [hunk 1]
```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    class="mobile-navigation clearfix"
+    :class="[
```
Comment (venimus) [inline src/components/layout/MobileNavigation.vue]: i would prefer using object
```
:class="{
   open: open
   sticky: $route.name !== 'home'
}"

File: src/components/layout/MobileNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    >
+      &#x2715;
+    </div>
+    <navigation/>
+    <div class="mobile-actions">
+      <img
+        class="tip"
+        :src="emptyHeart"
+        @click="toggleTipModal(true)"
+        v-if="!open && $route.name === 'home'"
```
Comment (venimus) [inline src/components/layout/MobileNavigation.vue]: you can move the v-if=!open part in the parent element, seems more logical to me

Comment (stanislav-slavov) [reply src/components/layout/MobileNavigation.vue]: The trigger-navigation button has a different condition. There are 2 buttons that disappear if   we are not on the home page. The parent would hide all and is used for styling.

File: src/components/layout/MobileNavigation.vue [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+      >
+      <img
+        class="trigger-navigation"
+        :src="iconMobileMenu"
+        @click="openNavigation(true)"
+        v-if="!open"
+      >
+    </div>
+    <div class="footer text-center" v-if="open">
```
Comment (venimus) [inline src/components/layout/MobileNavigation.vue]: please don't use text-center (or similar), there is a custom class already (.footer) just add centering there.

File: src/components/layout/MobileNavigation.vue [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      SuperHero is Open Source
+      <router-link class="ae" to="/terms">
```
Comment (venimus) [inline src/components/layout/MobileNavigation.vue]: can you add some specific class name to this, .ae is used elsewhere for styling the AE currency

File: src/components/layout/MobileNavigation.vue [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+      </router-link>
+    </div>
+    <mobile-send-tip-modal></mobile-send-tip-modal>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import Navigation from './Navigation.vue';
+import iconMobileMenu from '../../assets/iconMobileMenu.svg';
```
Comment (venimus) [inline src/components/layout/MobileNavigation.vue]: i think this import is useful only for dynamic assets. You can simply put that in src="" (without `:`).

File: src/components/layout/MobileNavigation.vue (hunk 2 above)
Comment (venimus) [reply src/components/layout/MobileNavigation.vue]: i mean only !open but leave the route === home per item where applies

Comment (stanislav-slavov) [reply src/components/layout/MobileNavigation.vue]: Yes, you are right

File: src/views/TipsList.vue [hunk 6]
```diff
@@ -73,10 +77,12 @@ import TipRecord from '../components/tipRecords/TipRecord.vue';
 import SendTip from '../components/layout/SendTip.vue';
 import LeftSection from '../components/layout/LeftSection.vue';
 import RightSection from '../components/layout/RightSection.vue';
+import MobileNavigation from '../components/layout/MobileNavigation.vue';
 import { EventBus } from '../utils/eventBus';
 import Loading from '../components/Loading.vue';
 import Onboarding from '../components/onboarding/Wizard.vue';
 import { MIDDLEWARE_URL } from '../config/constants';
+import iconEraser from '../assets/iconEraser.svg';
```
Comment (venimus) [inline src/views/TipsList.vue]: you missed one

Comment (stanislav-slavov) [reply src/views/TipsList.vue]: Thanks. Also I will restore the back arrow for other resolutions than mobile in the next commit

File: src/store/index.js [hunk 7]
```diff
@@ -30,6 +30,8 @@ export default new Vuex.Store({
     chainNames: [],
     wizardCurrentStep: 0,
     wizardIsCollapsed: false,
+    hideMobileNavigation: false,
```
Comment (venimus) [inline src/store/index.js]: do we need to persist the state of the navigation?

Comment (stanislav-slavov) [reply src/store/index.js]: It is implemented like so - if we have to collapse the mobile menu from a different pages and components. In this case we have the search.

File: src/views/CreateProfile.vue [hunk 8]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+    <mobile-navigation></mobile-navigation>
```
Comment (venimus) [inline src/views/CreateProfile.vue]: this needs to evolve in a Layout component some time in the near future

**Comment by mradkov** [comment]
Can you resolve the conflicts @stanislav-slavov ?

**Comment by stanislav-slavov** [comment]
> Can you resolve the conflicts @stanislav-slavov ?

Done.

**Comment by mradkov** [comment]
Still some conflicts.

**Comment by stanislav-slavov** [comment]
> Still some conflicts.

Rebased

**Pull request merged**